github/store/teststore.go

```diff
@@ -0,0 +1,117 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/src-d/metadata-retrieval/github/graphql"
+)
+
+type TestStore struct {
+	organization *graphql.Organization
+	repository   *graphql.RepositoryFields
+	topics       []string
+	users        []*graphql.UserExtended
+	prs          []*graphql.PullRequest
+	prComments   []*graphql.IssueComment
+}
+
+func (s *TestStore) GetRepository() *graphql.RepositoryFields {
+	return s.repository
+}
+
+func (s *TestStore) GetOrganization() *graphql.Organization {
+	return s.organization
+}
+
+func (s *TestStore) GetNumOfPrs() int {
+	return len(s.prs)
+}
+
+func (s *TestStore) GetNumOfPrComments() int {
+	return len(s.prComments)
+}
+
+func (s *TestStore) GetNumOfUsers() int {
+	return len(s.users)
+}
```

Comment:
I don't see that these methods would help a lot during the testing because it's still easy to access the fields